File: backend/legalsaathi/src/main/java/com/ascoder1109/legalsaathi/dtos/LoginResponseDTO.java
package com.ascoder1109.legalsaathi.dtos;

import lombok.Data;

@Data
public class LoginResponseDTO {
    private String token;
    private String message;
    private String name;
    private String role;

    public LoginResponseDTO(String token, String message,  String name, String role) {
        this.token = token;
        this.message = message;
        this.name = name;
        this.role = role;
    }
}

File: frontend/legalsaathi/src/app/core/dtos/LoginResponseDTO.ts
import { Role } from "../models/Role";

export class LoginResponseDTO {
    token: string;
    message: string;
    name: string;
    role: Role;

    constructor(token: string, message: string, name: string, role: Role) {
        this.token = token;
        this.message = message;
        this.name = name;
        this.role = role;
    }
}
